callisto/notification/api.py

```diff
@@ -57,8 +67,8 @@ def send_matching_report_to_school(cls, matches, identifier):
         cls.send_email_to_coordinator(pdf, 'match_delivery', report_id)
 
     @classmethod
-    def send_user_notification(cls, form, notification_name):
-        notification = cls.model.objects.on_site().get(name=notification_name)
+    def send_user_notification(cls, form, notification_name, site_id=None):
+        notification = cls.model.objects.on_site(site_id).get(name=notification_name)
```

Comment:
I do, since 99% of the tests work off of that case. The only ones that don't (and instead involve passing in a `site_id`) are in `tests/notification/test_sites.py::SiteRequestTest` https://github.com/SexualHealthInnovations/callisto-core/pull/151/files#diff-583950e0bb6e0a78fa9f5f2592038e20R61